mam4_refactor_scripts/test_gen_for_sub.sh

```diff
@@ -7,16 +7,16 @@ main() {
     #-------------------------
 
     #subroutine name
-    sub_name=clddiag
```

Comment:
Maybe we should just comment out the old `sub_name`, `module_name`, `dir_path`, and `cpp_directive` values instead of removing them, so we can revisit things if we have to.

Reply:
Thank you for the comment. This script is for the purpose of generating the templates of yaml files. These are input parameters of the script will change for each subroutine. it would be easy to revisit anyway. I think we don't have to keep all the old values.